src/common/commonOptionButton.module.css

```diff
@@ -0,0 +1,40 @@
+.button {
+   font-size: 1rem;
+   background-color: white;
+   border: 1px solid black;
+   border-radius: 0.7em;
+   cursor: pointer;
+   width: 100%;
+   height: 100%;
+   box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
+}
+
+.button.open {
+   border-bottom: none;
+   border-bottom-left-radius: 0;
+   border-bottom-right-radius: 0;
+}
+
+.optionsList {
+   background-color: white;
+   border: 1px solid #ccc;
+   border-bottom-left-radius: 0.7em;
+   border-bottom-right-radius: 0.7em;
+   box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
+   margin: 0;
+}
+
+.optionItem {
+   font-family: Roboto;
+   font-weight: 300;
+   height: 3em;
+   cursor: pointer;
+   transition: background-color 0.2s;
+   display: flex;
+   justify-content: center;
+   align-items: center;
+}
+
+.optionItem:hover {
+   background-color: #f0f0f0;
+}
```

Comment:
I love it that you understood that alot of your option buttons are going to use the same CSS so you made it a common style file!